### PR TITLE
Correct Terminology/slang

### PR DESCRIPTION
**Pull request by Kimax89**
Because no one collects Heroin and Coke straight of plants.

### CODE REVIEW

**Comment by Jawshy** [comment]
I noticed a few things off such as: 
- Raw Oil rather than Crude Oil. 
- Unused `STR_Item_Cocaine` (`STR_Item_CocaineU` and `STR_Item_CocaineP` are used.)
- You haven't changed it for the other languages.

File: Altis_Life.Altis/stringtable.xml [hunk 1]
```diff
@@ -3659,7 +3659,7 @@
       <Polish>Ropa naftowa</Polish>
     </Key>
     <Key ID="STR_Item_OilP">
-      <Original>Processed Oil</Original>
```
Comment (sargken) [inline Altis_Life.Altis/stringtable.xml]: It can be different types of Oil when its refined.

**Comment by danielstuart14** [comment]
hmmmmmm, going to take a look later...

**Comment by Heavybob** [comment]
Let me just consult my "specialist" Mr xX420_BlaZZ_IT_Doctor_FruitloopsXx on the right terminology.

**Pull request merged**